src/main/java/org/broadinstitute/consent/http/db/UserDAO.java

```diff
@@ -175,4 +175,12 @@ void updateUser(@Bind("displayName") String displayName,
             @BindList("datasetIds") List<Integer> datasetIds,
             @BindList("roleNames") List<String> roleNames);
 
+    @UseRowMapper(UserWithRolesMapper.class)
+    @SqlQuery("SELECT du.*, r.roleid, r.name, ur.dac_id " +
+            " FROM dacuser du " +
+            " LEFT JOIN user_role ur ON ur.user_id = du.dacuserid " +
+            " LEFT JOIN roles r ON r.roleid = ur.role_id " +
+            " WHERE du.institution_id = :institutionId")
+    List<User> findUsersByInstitution(@BindList("institutionId") Integer institutionId);
```

Comment:
Should just be a `@Bind` since it's a single integer. `BindList` is for collection types.